Branch-SDK/Branch-SDK/BNCLinkData.m

```diff
@@ -131,16 +115,6 @@ - (void)setupParams:(NSDictionary *)params {
     }
 }
 
-- (void)setObject:(id)anObject forKey:(id <NSCopying>)aKey {
```

Comment:
Isn't this method still called? Branch.m line 896, BranchShortUrlRequest.m line 79 and BranchShortUrlSyncRequest.m line 73?